nirodents/cli/brainextraction.py

```diff
@@ -91,6 +97,7 @@ def main():
     opts = get_parser().parse_args()
     update_templateflow(overwrite=False)
     be = init_rodent_brain_extraction_wf(
+        ants_affine_init=opts.antsai_init,
```

Comment:
Is this the right way around? If the default for `--no-antsAI` is `True`, then `ants_affine_init` is also `True`, so will carry out the workflow?

Reply:
Never mind, I worked it out. It stores `False` if the argument isn't present.

_However_, running this with the `--no-antsAI` command produced reports on Sherlock, so either the reports weren't generated because the antsAI call is so bad that they hang and then cancel, or it is something to do with when `ants_affine_init` is `True`

Reply:
setting ``--no-antsAI`` stores ``False`` in `opts.antsai_init` - so this is working :)